network-api/networkapi/buyersguide/templates/category_nav_links.html

```diff
@@ -1,8 +1,13 @@
 {% load bg_nav_tags %}
 
+{% comment %}
+  This is the new namespaced category_nav_links.html
+  This file is pending deletion when we delete the buyersguide app.
```

Comment:
this is the original file =)
```suggestion
  This file is pending deletion when we delete the buyersguide app.
```